homeassistant/components/nut/__init__.py

```diff
@@ -180,27 +193,34 @@ def __init__(self, host, port, alias, username, password):
         # Establish client with persistent=False to open/close connection on
         # each update call.  This is more reliable with async.
         self._client = PyNUTClient(self._host, port, username, password, 5, False)
+        self._ups_list = {}
         self._status = None
 
     @property
     def status(self):
```

Comment:
We don't need the property. We can just use an instance attribute directly.

Reply:
done